roles/README.md

```diff
@@ -14,6 +14,7 @@ looking for more responsibilities.
  - Software Engineer 2
  - [Senior Software Engineer](senior_software_engineer.md) – also looking for strong AWS experience for a [full stack role in Bristol](senior_software_engineer_with_aws.md)
  - [Lead Software Engineer](lead_software_engineer.md)
+ - [Principal Technologist](principal_technologist.md)
 
 You may have also seen that we have people in Tech Lead and Tech Architect roles. These are hats worn by appropriate Senior 1 and 2s.
```

Comment:
Nor directly related to this PR - should we also remove or modify the reference to S1/2s here?

Reply:
Good catch, thanks for that!

Reply:
Resolved in #373.